packages/ckb-sdk-core/examples/sendTransactionWithMultiplePrivateKey.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable */
+const Core = require('../lib').default
+const core = new Core('http://localhost:8114')
+
+const sk1 = '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee' // exmaple private key
+const sk2 = '0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff' // example private key
+
+const loadCells = async () => {
+  await core.loadSecp256k1Dep()
+  const lockScript1 = {
+    codeHash: core.config.secp256k1Dep.codeHash,
+    hashType: core.config.secp256k1Dep.hashType,
+    args: `0x${core.utils.blake160(core.utils.privateKeyToPublicKey(sk1), 'hex')}`
+  }
+
+  const lockScript2 = {
+    codeHash: core.config.secp256k1Dep.codeHash,
+    hashType: core.config.secp256k1Dep.hashType,
+    args: `0x${core.utils.blake160(core.utils.privateKeyToPublicKey(sk2), 'hex')}`
+  }
+
+  const lockHash1 = core.utils.scriptToHash(lockScript1)
+  const lockHash2 = core.utils.scriptToHash(lockScript2)
+
+  await core.loadCells({
```

Comment:
Moving `loadCells` from core to example will be more better ?

Reply:
Let user determine the collecting live cells strategy and we support one strategy as a reference

Reply:
> Let user determine the collecting live cells strategy and we support one strategy as a reference

It's added here for users to watch the process clearer, they can uncomment
```javascript
// loadCells()
// generateTransaction()
// sendTransaction()
```
to see the three steps in sending transactions.

Reply:
I mean `loadCells` of core will give user a mistake which is recommended officially and move it to example will avoid the mistake.

Reply:
> I mean `loadCells` of core will give user a mistake which is recommended officially and move it to example will avoid the mistake.

It's used as default searching strategy in many methods, I'll remove them in another PR.